raml/api-reader.raml

```diff
@@ -0,0 +1,72 @@
+#%RAML 0.8
+title: Read Only Runbook API Service
+version: v1
+mediaType: application/json
+baseUri: /api/{version}
```

Comment:
yeah, looks like you're right. I just wanted to use it as a shorthand, will remove